src/AaveV2Update_20231009/DeployAaveV2Update_20231009.s.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {GovHelpers} from 'aave-helpers/GovHelpers.sol';
+import {
+  EthereumScript
+} from 'aave-helpers/ScriptUtils.sol';
+import {
+  AaveV2EthereumUpdate20231009Payload
+} from './AaveV2Ethereum_20231009.sol';
+
+contract CreateProposal is EthereumScript {
+  function run() external broadcast {
+    GovHelpers.Payload[] memory payloads = new GovHelpers.Payload[](1);
+    payloads[0] = GovHelpers.buildMainnet(address(0));
+    GovHelpers.createProposal(
+      payloads,
+      GovHelpers.ipfsHashFile(
+        vm,
+        'src/AaveV2Update_20231009/AaveV2Update_20231009.md'
```

Comment:
the naming convention of this is not in line with the generator. It's a bit annoying as "new proposals" are usually correctly sorted due to YYYYMMDD folder prefix, but this one is not.

That said, I guess it's acceptable as there#s some inconsistency with old proposals anyways, but in the future plz give the generator a shot which will correctly generate structures & aip references.